### PR TITLE
Use new social banner

### PR DESCRIPTION
**Pull request by goetas**
Currently it looks that the https://si0.twimg.com/profile_images/3735529170/0107684b9ce550ecf60ad86a9f104bac_bigger.png image is not available (twitter changed the host name to `pbs.twimg.com`).

Anyway that image was very small, so in the tweet there was no logo, see:
image

I have created a new social banner image (following the Github [guidelines](https://help.github.com/en/articles/customizing-your-repositorys-social-media-preview)) and made this PR.

Did not have a way to test it, but it should work. Anyway even the current version does not work :)

### CODE REVIEW

**Comment by woodworker** [comment]
thanks for the change

**Comment by goetas** [comment]
:( something did not work

Still not displayed correctly

File: _layouts/default.html [hunk 1]
```diff
@@ -14,16 +14,16 @@
...
+        <meta property="og:image" content="//www.bephpug.de/assets/social-banner.png">
+        <meta name="og:description" content="We meet on the 1st tuesday each month at Co-Up to talk about PHP, Web-development and related topics."/>
 
         <meta name="Twitter:card" content="summary">
         <meta name="Twitter:site" content="@bephpug">
         <meta name="Twitter:creator" content="@bephpug">
         <meta name="Twitter:title" content="{% if page.title %}{{ page.title }} - Berlin PHP Usergroup{% else %}Berlin PHP Usergroup{% endif %}">
         <meta name="Twitter:description" content="We meet on the 1st tuesday each month at Co-Up to talk about PHP, Web-development and related topics.">
-        <meta name="Twitter:image" content="//si0.twimg.com/profile_images/3735529170/0107684b9ce550ecf60ad86a9f104bac_bigger.png">
+        <meta name="Twitter:image" content="//www.bephpug.de/assets/social-banner.png">
```
Comment (goetas) [inline _layouts/default.html]: I guess that Twitter does not like the `//` without the protocol

**Comment by goetas** [comment]
I guess that Twitter does not like the // without the protocol

**Comment by Flyingmana** [comment]
I can second that, twitter needs absolute paths here

**Pull request merged**